.github/workflows/ci.yml

```diff
@@ -420,4 +420,4 @@ jobs:
           turbo-token: ${{ secrets.TURBO_TOKEN }}
 
       - name: Publish with pkg-pr-new
-        run: pnpm run build && pnpx pkg-pr-new publish --compact --pnpm './packages/*'
+        run: pnpm run build && pnpx pkg-pr-new@${{ vars.PKG_PR_NEW_VERSION || 'latest' }} publish --compact --pnpm './packages/*'
```

Comment:
Can we replace `'latest'` with the current version? As it stands now if we didn't have the env var defined, we'd be loading the latest version. Given that only certain folks on the team have env var permission, it's possible that we'd be silently defaulting to `latest` without knowing it. I'm okay with using env var provided there's a fallback to a specific known version.